src/main/java/mate/controller/CarController.java

```diff
@@ -0,0 +1,70 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+public class CarController extends HttpServlet {
```

Comment:
```suggestion
public class AddCarController extends HttpServlet {
```
the same in other controllers that create driver and manufacturer